aam.php

```diff
@@ -1,3 +1,47 @@
+<?php
+
+require 'core.inc.php';
+require 'connect.inc.php';
+if (!loggedin()) {
+
+} else{
+        $name=getfield('name');
+        header('location:home.php');
+}
+?>
+<?php 
+
+if (isset($_POST['email'])&&isset($_POST['password'])) {
+  $email=$_POST['email'];
```

Comment:
Clean input before processing.
http://stackoverflow.com/questions/129677/whats-the-best-method-for-sanitizing-user-input-with-php
http://php.net/manual/en/pdo.prepared-statements.php